patterns/1-initial/extensions-for-sustainable-growth.md

```diff
@@ -54,17 +70,19 @@ In order for the extensions model to be successful, there are few architectural
    - Create criteria for when an extension can be ported to the primary repository, such as adoption of the extension by internal products and demand for the feature.
    - Porting process of the extension to the primary repository will follow more stringent code review guidelines as set by library maintainers.
 
+![Project extensions](../../assets/img/extensions-for-sustainable-growth/extensions-for-sustainable-growth.png)
+
 Following the principles outlined above ensures that:
 
-- The addition of the extensions pattern with easy to use templates enables developers to add new features to a project's ecosystem without requiring them to write large amounts of boilerplate code.
+- The addition of the extensions pattern with easy to use templates enables developers to add new features to a project's ecosystem without requiring them to write large amounts of [boilerplate](https://en.wikipedia.org/wiki/Boilerplate_code) code.
```

Comment:
```suggestion
- Developers can add new features to a project's ecosystem without requiring them to write large amounts of [boilerplate](https://en.wikipedia.org/wiki/Boilerplate_code) code.
```

I would not talk about the "pattern" at this point, as that is not how they users would think about it.
They might think "I added an extension using a template" but probably not "I used the extension pattern" :)